src/test/java/org/broadinstitute/consent/http/db/DAOTestHelper.java

```diff
@@ -265,4 +274,23 @@ private DataAccessRequest createDataAccessRequest(Integer userId, Integer collec
     return dataAccessRequestDAO.findByReferenceId(referenceId);
   }
 
+  protected Map<String, FormDataBodyPart> getRandomFiles(Integer count) {
+    Map<String, FormDataBodyPart> mapOfFiles = new HashMap<>();
+    IntStream.range(0, count)
+        .forEach(index -> {
+          String name = String.format("file%d", index);
+          mapOfFiles.put(name, getFormDataBodyPartMock(name));
+        });
+    return mapOfFiles;
+  }
+
+  private FormDataBodyPart getFormDataBodyPartMock(String name) {
+    FormDataBodyPart part = mock(FormDataBodyPart.class);
+    lenient().when(part.getName()).thenReturn(name);
+    lenient().when(part.getMediaType()).thenReturn(MediaType.MULTIPART_FORM_DATA_TYPE);
+    lenient().when(part.getValueAs(InputStream.class))
+        .thenReturn(new ByteArrayInputStream(EMPTY_JSON_DOCUMENT.getBytes()));
```

Comment:
I think we should avoid the use of `lenient` to ensure that we have strict mocks.

Reply:
They're gone!  I think we have one other spot related to library cards that are not part of this PR.  See: src/test/java/org/broadinstitute/consent/http/service/LibraryCardServiceTest.java (strictness = Strictness.LENIENT)

Should I make a ticket to refactor so this can be removed?